dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/merger/ArrayMerger.java

```diff
@@ -25,34 +26,48 @@
     public static final ArrayMerger INSTANCE = new ArrayMerger();
 
     @Override
-    public Object[] merge(Object[]... others) {
-        if (others.length == 0) {
-            return null;
+    public Object[] merge(Object[]... items) {
+        if (ArrayUtils.isEmpty(items)) {
+            return new Object[0];
         }
+
+        int i = 0;
```

Comment:
why check this?

Reply:
Because if the user pass in (   null ,null , new String[]{"abs","edf"} , null   ) ,  we need to find out the  String.class  , so we need to skip the null objects .